### ISSUE TITLE
Tax Declaration issues (ES)

### ISSUE THREAD

**Issue by anarosa-es** [SEMICeu/SDG-sandbox#39]
We assume that this "tax declaration" is about natural persons, so we are talking about our IRPF data service that provides information on the level of income of a given person.
1. we guess that "**paymentDate**" is the tax year. If so, we think that the attribute should be called **"taxYear" for clearity**.
2. we would **need a clear definition of "totalIncome"**. We have a formula to calculate the income level that our data service provides, but we need to check is this concept is equivalent in any country. 
3. we guess that "incomeTax" is the amount to pay or to receive as result of the declaration. If so, **Why is "incomeTax" relevant** for cross-border public services? **We don't provide the amount to pay or receive by the tax subject**.
4. **we don't provide the "address"** of the tax payer 

In general, here you are other comments:
1. we think that "certificate" is something related only to documents. So we propose to **use the word "evidence"** instead which is the word used by the SDGR
2. we think that the **issuing attributes of an evidence should be present independently of the evidence**
3. as you know, we have 1st and 2nd family names that can be joint in a "family name" attribute, but for the legal validity of the data, we should be **able to specify the Spanish two family names as well**
4. we think that it is **necessary to model as well the input parameters for requesting a particular evidence**. This is important for data modelling and matching. Requesting an Spanish birth evidence by citizen. **The request of this data service has to include** the corresponding tax year and the next payer's data: Spanish fiscal or personal identification number (NIF or DNI), the full name, the given name, the 1st family name and the 2nd family name if exists.

**Comment by sethvanhooland** [comment]
Thank you Ana Rosa for the comments in regards to the Tax Declaration data model, we'll make sure to address these during the webinar. We will also take note of the other, more general comments on terminology, the usage of double family names and the question in regards to the input parameters you are referring to.

**Comment by cbahim** [comment]
Thanks @anarosa-es for raising these points. 

With regards to your specific comments:

1. We have created a separate issue #67  to discuss the dates related to a tax declaration certificate. 
2. We have created a separate issue #68  to clarify what is understood with “total income” by the different MS.
3. Income tax is optional, so no problem there.
4. The address of the tax payer is also optional, so no problem there.
